package com.comviva.day3;

public class Exception01 {
	public static void main(String[] args) {
		int res;
		int num1 = 10;
		int num2 = 0;
		try{
			int arr[] = new int[-4];
			res = num1/num2;
			System.out.println("Result is " + res);
		} catch(Exception e){
			System.out.println(e);
		}
		System.out.println("stuff....");
	}
}
